upath/implementations/hdfs.py

```diff
@@ -10,6 +10,15 @@ def touch(self, path, **kwargs):
         kwargs.pop("truncate", None)
         super().touch(path, **kwargs)
 
+    def mkdir(self, path, create_parents=True, **kwargs):
+        pth = self._format_path(path)
+        if create_parents:
+            return self._fs.makedirs(pth, **kwargs)
+        else:
+            if not kwargs.get("exist_ok", False) and self._fs.exists(pth):
```

Comment:
why not put `exist_ok` explicitly in the function signature ?

Reply:
I'm using the function signature defined in the base accessor cls previously defined in https://github.com/fsspec/universal_pathlib/pull/59

I don't think it's worth cleaning this up a lot, since (1) it's a private interface and (2) all of this has to be refactored again once the remaining pathlib changes land in python=3.12.